lib/constellation.rb

```diff
@@ -0,0 +1,39 @@
+class ConstellationHttpException < StandardError
+end
```

Comment:
Nice custom exception definition.

However, this isn’t going to let you [rescue any exceptions in your controller](https://github.com/artsy/rsvp-rails/pull/27/files#diff-8f333f5a407b28ce62f67c1aa85d6fffR16) that are raised from _inside_ `Net::HTTP`, if that’s your intention too?